bridges/FicbookBridge.php

```diff
@@ -135,8 +135,8 @@ private function collectUpdatesData($html)
             ];
 
             if ($this->getInput('include_contents')) {
-                $content = getSimpleHTMLDOMCached($item['uri']);
-                $item['content'] = $content->find('#content', 0);
+                $content = getSimpleHTMLDOMCached($item['uri'], 86400, [], [], true, true, DEFAULT_TARGET_CHARSET, false);
+                $item['content'] = str_replace("\n", '<br>', $content->find('#content', 0)->innertext);
```

Comment:
nl2br

Reply:
Thank you, I will know. I planning to fix getting single chapter later and than change str_replace to nl2br.